_std/color.dm

```diff
@@ -274,6 +274,29 @@ proc/hsv_transform_color_matrix(h=0.0, s=1.0, v=1.0)
 		0, 0, 0, 0
 	)
 
+proc/get_average_color(icon/I, xPixelInterval = 4, yPixelInterval = 4)
```

Comment:
I think the default should be 1 and not 4 here.

Reply:
doing 1000 checks seems a bit intense for general usage, doesn't it?